charts/drone/Chart.yaml

```diff
@@ -4,6 +4,7 @@ name: drone
 description: Drone is a self-service Continuous Delivery platform for busy development teams
 # TODO: Un-comment once we move back to apiVersion: v2.
 # type: application
+version: 0.6.3
```

Comment:
@coryjohnson-flume you added a new line with `version` rather than change the existing line